styles/components/_promo.scss

```diff
@@ -25,25 +29,25 @@
   color: $sprk-promo-subtitle-font-color;
 }
 
-/// Adds styles for the image
-/// in the Flag Promo variant.
-.sprk-c-Promo__image--flag {
+.sprk-c-Promo__image {
+  width: 100%;
+  height: 100%;
   object-fit: cover;
-  padding: $sprk-space-inset-m;
 
   @media (max-width: $sprk-promo-breakpoint) {
-    max-width: $sprk-promo-image-narrow-max-width;
     max-height: $sprk-promo-image-narrow-max-height;
   }
 }
 
-.sprk-c-Promo__image {
-  width: 100%;
-  height: 100%;
-  object-fit: cover;
+/// Adds styles for the image
+/// in the Flag Promo variant.
+.sprk-c-Promo__image--flag {
```

Comment:
We had it setup for flag images in Promo that you can only use the
`sprk-c-Promo__image--flag class`. This is incorrect and should be "`sprk-c-Promo__image  sprk-c-Promo__image--flag`". The `--flag` class doesnt utitilize the base settings on the original class and it should. If you add both like our naming convention would suggest, then the component breaks. We had a var `$sprk-promo-image-narrow-max-width` that we were only using on the flag class and not the regular one. So I renamed it to be flag so it is more clear what it is for. This is a breaking change for our frameworks for the Flag promo variant.